### ISSUE TITLE
New Youtube video about income sources and investors

### ISSUE THREAD

**Issue by quinnliu** [WalnutiQ/wAlnut#214]
Investors will have their names displayed on the main README.md sorted from greatest to least

**Comment by quinnliu** [comment]
Step 1) Set up bank of america core checking account
Step 2) Set up venmo
Step 3) Transfer youtube ad revenue up to day to this account
Step 4) change which account youtube makes it's deposits to
Step 5) make Youtube video about this change and how any viewer can become an investor in wAlnut

**Comment by quinnliu** [comment]
Wrote a new section in readme.md here: https://github.com/WalnutiQ/wAlnut#investors

**Comment by quinnliu** [comment]
UPDATE: Step 1 done! FUCK YA

**Comment by quinnliu** [comment]
### Step 2
First name: wAlnut
Last name: iQ

**Comment by quinnliu** [comment]
### Step 3

**Comment by quinnliu** [comment]
### Step 4
Super annoying need to come back after $100 plus in account.
TODO: need to call YouTube

**Comment by quinnliu** [comment]
### Step 5
done!

**Comment by quinnliu** [comment]
### Step 6
Many people have told me to quite doing this including my dad, mom, best friends, and even myself. And I did quite for about 1 year. But right now this feels very right. I'm currently working with 3 other people paying them nothing but I need to start paying them for their time and to make this project sustainable. You need both intrinsitic and extrinsitc motivation to keep doing something your curious about.

**Comment by quinnliu** [comment]
I don't know why but right now it doesn't feel right to make a video asking for donations. Closing this for now.

**Comment by madhephaestus** [comment]
I disagree, i think asking for $5k on indiegogo or something like that would be easily achievable. We could make a video of flashy things to raise money for development. If we keep the funding level low and the deliverable only software we would do anyway, maybe we could get a few months of operating expenses donated, and more importantly, lots of buzz and a wave of extra developers and users. Lots of people want something like this to exist, the problem is just that no one knows its here. 

just my 2 cents...

**Comment by quinnliu** [comment]
@madhephaestus I love the optimism Kevin, but I don't think people will be too willing to donate to something that may take another 10 years to finish building. I think those sites like kickstarter and indiegogo are more for products that can be in the consumers hands in the next year max. I am still debating about asking for money from the people that watch my YouTube videos. Most of them are broke college kids in crazy amounts of debt so it feels wrong for me to go ask them for money. That's why I've hesitated about making a donation video.

**Comment by quinnliu** [comment]
I've thought about it some more I think it will be good to make videos summarizing our progress and direction when I feel like there is something important to say. Like 10 minute videos. Then at the end I will show how easy it is to donate to the project. This way I am teaching our investors exactly what we are researching.

**Comment by madhephaestus** [comment]
Hey, have you looked into https://www.patreon.com/ ? Might be a nice place to gain steady support. Also, cross post it on https://gratipay.com/ ?

**Comment by quinnliu** [comment]
I think the venmo thing is good for now. And maybe when my youtube channel is bigger patreon will work. I only have 4k subs and those people subscribed for my programming videos which I don't make anymore.

**Comment by quinnliu** [comment]
I'm closing this for now so we can focus on building the algorithm. I honestly just wanted to pay out the few hundred dollars I made from Youtube to you guys to say thanks for helping me out on this crazy project. But in reality I don't feel comfortable asking for money until we have built something actually useful. This will take at least 5 more years. I am already very grateful for having the time to even work on this project and I'm expecting to work on this project for free for the rest of my life if it takes that long
